### ISSUE TITLE
Addons can’t be selected in backup

### ISSUE THREAD

**Issue by LEJOUI** [Sebclem/hassio-nextcloud-backup#160]
**Describe the bug**
I can’t select which addons I want to include in the backup. Also the backups aren’t created with the addons at this point
![BBA44F96-5802-48AC-A53D-22F92B43A488](https://user-images.githubusercontent.com/13658422/177029332-a5c2090b-b0df-42e1-a126-a8bbb68781fb.png)

**To Reproduce**
Open the backup settings in the Nextcloud-backup addon

**Expected behavior**
I’d expect to choose which addons I can select.

**System information**
- Home Assistant Version : 2021.6.7
- Home Assistant Supervisor Version : 8.2
- Nextcloud Version : 24.0.2
- Addon version : 0.17.3
- If any, reverse proxy in fort on Nextcloud : nginx proxy

**Additional context**

Logs are fine:

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] 10-run: applying... 
[fix-attrs.d] 10-run: exited 1.
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
 -----------------------------------------------------------
 Add-on: Nextcloud Backup
 Easily upload your Home Assistant backups to Nextcloud

 -----------------------------------------------------------
  Add-on version: 0.17.3
 You are running the latest version of this add-on.
 System: Home Assistant OS 8.2  (amd64 / qemux86-64)
 Home Assistant Core: 2022.6.7
 Home Assistant Supervisor: 2022.06.2
 -----------------------------------------------------------
Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
 -----------------------------------------------------------
 [cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to DEBUG
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[09:03:46] INFO: Starting Node...
[2022-07-03 09:03:55] [info]: 	Log level: debug
[2022-07-03 09:03:55] [info]: 	Backup timeout: 5400000
[2022-07-03 09:03:55] [info]: 	Satus : Go !
[2022-07-03 09:03:55] [info]: 	Initializing and checking webdav client...
[2022-07-03 09:03:55] [info]: 	Starting auto clean cron...
[2022-07-03 09:03:55] [info]: 	Starting Cron...
[2022-07-03 09:03:56] [info]: 	Hassio API : Go !
[2022-07-03 09:03:56] [debug]: 	Nextcloud connection:  Success !
[2022-07-03 09:03:57] [debug]: 	Path /Hassio Backup/ already exist.
[2022-07-03 09:03:57] [info]: 	Nextcloud connection : Go !

```

**Comment by dansharpy** [comment]
Ive also just noticed this issue happening to me as well. No add ons listed or included in the backup

**Comment by Sebclem** [comment]
Sorry for not replying before, I was stuck with COVID.

I can reproduce this, I will try to fix this quickly.

**Comment by LEJOUI** [comment]
No need to apologies, it’s all in free time made. Hope you’re doing well.
We’ll see the fido coming when you’ve found the time. :)

**Comment by Sebclem** [comment]
Fixed with a3f92e4 , I just have issue to push the docker images, I will retry tomorrow :+1:

**Comment by Sebclem** [comment]
It's now fixed and deployed with [0.17.6](https://github.com/Sebclem/hassio-nextcloud-backup/releases/tag/0.17.6)
Re-open if the issue is still present 👍
